main/main.ino

```diff
@@ -401,6 +401,7 @@ void connectMQTT() {
   char topic[mqtt_topic_max_size];
   strcpy(topic, mqtt_topic);
   strcat(topic, will_Topic);
+  client.setBufferSize(mqtt_max_packet_size);
```

Comment:
ok for this change,

below you can find also a warning 
```
#if defined(ESP8266) || defined(ESP32) || defined(__AVR_ATmega2560__) || defined(__AVR_ATmega1280__)
  if (MQTT_MAX_PACKET_SIZE == 128)
    Log.error(F("WRONG PUBSUBCLIENT LIBRARY USED PLEASE INSTALL THE ONE FROM RELEASE PAGE" CR));
#endif
```
I think it could be removed and the trace just before log `mqtt_max_packet_size` instead of `MQTT_MAX_PACKET_SIZE `